DNN Platform/Website/App_GlobalResources/GlobalResources.resx

```diff
@@ -826,35 +801,7 @@ Thank you,
-            &lt;li&gt;&lt;strong&gt;If using Windows 2000 - IIS5&lt;/strong&gt;
-		    &lt;br/&gt;
-		    &lt;ol style="list-style-type: circle;"&gt;
-		      &lt;li&gt;the [SERVER]\ASPNET User Account must have Read, Write, and Change Control of the virtual root of your website.&lt;/li&gt;
-		    &lt;/ol&gt;
-		    &lt;br/&gt;
-		  &lt;/li&gt;
-            &lt;li&gt;&lt;strong&gt;If using Windows 2003 - IIS6 or Vista/Windows 2008 - IIS7&lt;/strong&gt;
-		    &lt;br/&gt;
-		    &lt;ol style="list-style-type: circle;"&gt;
-		      &lt;li&gt;the NT AUTHORITY\NETWORK SERVICE User Account must have Read, Write, and Change Control of the virtual root of your website.&lt;/li&gt;
-		    &lt;/ol&gt;
-		  &lt;/li&gt;
-		&lt;/ul&gt;
-        &lt;p&gt;&lt;img alt="Image showing the security settings for the root folder of the website must be set to Read, Write and Change Control for the ASPNET or NT AUTHORITY/NETWORK SERVICE accounts." src="images/403-3.gif"&gt;&lt;/p&gt;
-  &lt;/div&gt;</value>
+    <value>&lt;div style="height: 100%; width: 90%;"&gt;     &lt;h3&gt;DNN Configuration Error&lt;/h3&gt;     &lt;p&gt;{0}&lt;/p&gt;     &lt;p&gt;Your DNN installation appears to have a security settings problem (most likely permissions). DNN has extensive file upload capabilities for content, modules, and themes. These features require custom security settings so that the application is able to create and remove files in your website.&lt;/p&gt;     &lt;p&gt;On your webserver, browse to the root folder of the website (C:\DotNetNuke by default) using Windows Explorer. Right-click the folder and select "Sharing and Security" from the popup menu. Select tab "Security". Add the appropriate user account and set proper permissions for the ASP.Net user, specified in advanced setting of the IIS application pool that is hosting your website.&lt;/p&gt;     &lt;ul style="text-align: left; list-style-type: none; list-style-position: outside;"&gt;         &lt;li&gt;&lt;strong&gt;If using a named account like builtin "NT AUTHORITY\NETWORK SERVICE"&lt;/strong&gt;&lt;br/&gt;             &lt;ol style="list-style-type: circle;"&gt;                 &lt;li&gt;The user Account must be granted Read, Write, Read &amp; Execute, and Modify permission.&lt;/li&gt;             &lt;/ol&gt;             &lt;br/&gt;         &lt;/li&gt;         &lt;li&gt;&lt;strong&gt;If using "ApplicationPoolIdentity" (recommended)&lt;/strong&gt;&lt;br/&gt;             &lt;ol style="list-style-type: circle;"&gt;                 &lt;li&gt;User account IIS AppPool\[AppPool Name] (replace [AppPool Name] by the real name of the ISS application pool) must be granted Read, Write, Read &amp; Execute, and Modify permission.&lt;/li&gt;            &lt;/ol&gt;         &lt;/li&gt;     &lt;/ul&gt;     &lt;p&gt;&lt;img alt="Image showing the security settings for the root folder of the website must be set to Read, Write and Change Control for a named account." src="images/403-3.gif"&gt;&lt;/p&gt; &lt;/div&gt;</value>
```

Comment:
```suggestion
    <value>&lt;div style="height: 100%; width: 90%;"&gt;     &lt;h3&gt;DNN Configuration Error&lt;/h3&gt;     &lt;p&gt;{0}&lt;/p&gt;     &lt;p&gt;Your DNN installation appears to have a security settings problem (most likely permissions). DNN has extensive file upload capabilities for content, modules, and themes. These features require custom security settings so that the application is able to create and remove files in your website.&lt;/p&gt;     &lt;p&gt;On your webserver, browse to the root folder of the website (C:\DotNetNuke by default) using Windows Explorer. Right-click the folder and select "Sharing and Security" from the popup menu. Select tab "Security". Add the appropriate user account and set proper permissions for the ASP.Net user, specified in advanced setting of the IIS application pool that is hosting your website.&lt;/p&gt;     &lt;ul style="text-align: left; list-style-type: none; list-style-position: outside;"&gt;         &lt;li&gt;&lt;strong&gt;If using a named account like builtin "NT AUTHORITY\NETWORK SERVICE"&lt;/strong&gt;&lt;br/&gt;             &lt;ol style="list-style-type: circle;"&gt;                 &lt;li&gt;The user Account must be granted Read, Write, Read &amp; Execute, and Modify permission.&lt;/li&gt;             &lt;/ol&gt;             &lt;br/&gt;         &lt;/li&gt;         &lt;li&gt;&lt;strong&gt;If using "ApplicationPoolIdentity" (recommended)&lt;/strong&gt;&lt;br/&gt;             &lt;ol style="list-style-type: circle;"&gt;                 &lt;li&gt;User account IIS AppPool\[AppPool Name] (replace [AppPool Name] by the real name of the IIS application pool) must be granted Read, Write, Read &amp; Execute, and Modify permission.&lt;/li&gt;            &lt;/ol&gt;         &lt;/li&gt;     &lt;/ul&gt;     &lt;p&gt;&lt;img alt="Image showing the security settings for the root folder of the website must be set to Read, Write and Change Control for a named account." src="images/403-3.gif"&gt;&lt;/p&gt; &lt;/div&gt;</value>
```